RELEASE_NOTES.md

```diff
@@ -77,6 +77,13 @@ v1.0.0-alpha.xx
   chance to clean up between tests.
   [#994](https://github.com/OpenAssetIO/OpenAssetIO/issues/994)
 
+- `close` method added to `EntityReferencePagerInterface` for managers
+  to override in order to respond to hosts closing paging queries.
+  This method is now called upon `EntityReferencePager` destruction,
+  supplanting the prior recommended method of reacting in the
+  destructor/del method of `EntityReferencePagerInterface`
+  [#990](https://github.com/OpenAssetIO/OpenAssetIO/issues/990)
```

Comment:
[fix] We can remove this, as we never released with the old approach (see above).

Reply:
Oh right yeah, nice catch.

https://github.com/OpenAssetIO/OpenAssetIO/pull/1013/commits/1dc2d9f3c77226dfd026ed2c40055c829787eeb2

Reply:
Resolving on behalf of Tom since he's away